src/main/java/online/book/store/repository/imp/BookRepositoryImpl.java

```diff
@@ -0,0 +1,54 @@
+package online.book.store.repository.imp;
+
+import java.util.List;
+import online.book.store.model.Book;
+import online.book.store.repository.BookRepository;
+import org.hibernate.Session;
+import org.hibernate.SessionFactory;
+import org.hibernate.Transaction;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Repository;
+
+@Repository
+public class BookRepositoryImpl implements BookRepository {
+    private static final String CANT_SAVE_MSG = "Can't save book: ";
+    private static final String CANT_GET_ALL_MSG = "Can't get all books!";
+    private static final String SELECT_ALL_QUERY = "SELECT a FROM Book a";
```

Comment:
Use full word or use first entity name letter.
```suggestion
    private static final String SELECT_ALL_QUERY = "SELECT b FROM Book b";
```